### PR TITLE
Make dropdown and search bootstrap on editor page

### PR DESCRIPTION
**Pull request by liamhickey**
#102

### CODE REVIEW

File: Project/src/main/resources/templates/editor.html [hunk 1]
```diff
@@ -6,11 +6,12 @@
...
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
-    <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.js"></script>
+    <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
+    <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.18/js/dataTables.bootstrap4.min.js"></script>
     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/tempusdominus-bootstrap-4/5.0.1/js/tempusdominus-bootstrap-4.min.js"></script>
     <script src="/js/editor.js"></script>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tempusdominus-bootstrap-4/5.0.1/css/tempusdominus-bootstrap-4.min.css" />
-    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/jquery.dataTables.css">
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css">
```
Comment (br0wncrust4cean) [inline Project/src/main/resources/templates/editor.html:14]: is there no webjars import from maven for all of this?

**Review by br0wncrust4cean — CHANGES_REQUESTED** [review]

File: Project/src/main/resources/templates/editor.html (hunk 1 above)
Comment (Taz-Z) [reply Project/src/main/resources/templates/editor.html:14]: The webjar pull has a well known error with data tables

Comment (Taz-Z) [reply Project/src/main/resources/templates/editor.html:14]: I mentioned it in a pr awhile back, highlights the first row for no reason

**Review by Taz-Z — APPROVED** [review]

File: Project/src/main/resources/templates/editor.html (hunk 1 above)
Comment (br0wncrust4cean) [reply Project/src/main/resources/templates/editor.html:14]: oh true

**Review by br0wncrust4cean — APPROVED** [review]

**Comment by liamhickey** [comment]
Resolves #102

**Pull request merged**